dojo/api_v2/serializers.py

```diff
@@ -1531,13 +1532,27 @@ def get_engagement(self, obj):
         )
 
     def validate(self, data):
+        findings = data.get('accepted_findings', [])
+        findings_ids = list(map(lambda x: x.id, findings))
+        finding_objects = Finding.objects.filter(id__in=findings_ids)
+        authed_findings = get_authorized_findings(Permissions.Finding_View).filter(id__in=findings_ids)
```

Comment:
```suggestion
        authed_findings = get_authorized_findings(Permissions.Finding_Edit).filter(id__in=findings_ids)
```
Setting a risk exception should likely require at least edit access to a finding. Also removing extra var